.github/workflows/bump-versions.yaml

```diff
@@ -0,0 +1,25 @@
+name: bump-versions
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The new version
+        required: true
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # write for peter-evans/create-pull-request, read for actions/checkout
+      pull-requests: write # write for peter-evans/create-pull-request
+    steps:
+      - run: |
+          mvn versions:set -DgenerateBackupPoms=false -DnewVersion=${{ github.event.inputs.version }}
+      - uses: peter-evans/create-pull-request@v3.5.1
```

Comment:
The newest version is `5.0.2`. Any reason to use the old one?

Reply:
I must've copied it from somewhere. Updated to the latest